forge-gui/res/cardsfolder/upcoming/goldwardens_gambit.txt

```diff
@@ -0,0 +1,10 @@
+Name:Goldwardens' Gambit
+ManaCost:6 R R
+Types:Sorcery
+K:Affinity:Equipment
+A:SP$ Token | TokenScript$ r_2_2_rebel | TokenOwner$ You | RememberTokens$ True | TokenAmount$ 5 | SubAbility$ DBRepeat | PumpKeywords$ Haste | PumpDuration$ EOT | SpellDescription$ Create five 2/2 red Rebel creature tokens. They gain haste until end of turn. For each of those tokens, you may attach an Equipment you control to it.
+SVar:DBRepeat:DB$ RepeatEach | UseImprinted$ True | DefinedCards$ DirectRemembered | RepeatSubAbility$ DBAttach
+SVar:DBAttach:DB$ Attach | Choices$ Equipment.YouCtrl | Defined$ Imprinted | ChoiceTitle$ Select an equipment to attach to this token | Optional$ True
```

Comment:
> You choose which Equipment to attach to each token, then attach those Equipment all at once. You can't move the same Equipment around to each of the tokens as the ability resolves.

Simultaneous is probably too difficult currently but I think you can use `RememberAttached$` so it can't be abused too easily

Reply:
does it need Cleanup now?